solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -9,12 +9,20 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 library MinimumStakeSchedule {
     using SafeMath for uint256;
 
+    // Apr-28-2020 02:52:46 AM UTC when KEEP token has been deployed
+    // TX:  0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014
+    uint256 public constant scheduleStart = 1588042366;
+    
     // 2 years in seconds (seconds per day * days in a year * years)
     uint256 public constant schedule = 86400 * 365 * 2;
     uint256 public constant steps = 10;
     uint256 public constant base = 10000 * 1e18;
 
-    function current(uint256 scheduleStart) public view returns (uint256) {
+    function current() public view returns (uint256) {
```

Comment:
since this is `public` and the one below is `internal` maybe we can replace it with 1 function, ie:
```
function current() public view returns (uint256) {
        if (now < scheduleStart.add(schedule)) {
            uint256 currentStep = steps.mul(now.sub(scheduleStart)).div(schedule);
            return base.mul(steps.sub(currentStep));
        }
        return base;
    }
```

Reply:
Negative, please see the minimum stake schedule tests. We can't go back in time in tests, that hurts.

Reply:
I realize in another conversation a note about contract size came up, but it's worth fully articulating here would be that this library should "ideally" focus on the computation, and the staking contract should provide the start time, as was the case before. That also allows test flexibility, since tests can modify that start time to their heart's content, via stub. 

So it's strictly token staking contract size limitations (as I understand it) that have put us into this slightly odd setup. Correct me if I'm wrong, just trying to make sure the reasoning is fully captured.